_includes/sponsor-footer.html

```diff
@@ -1,5 +1,11 @@
 <div class="partner-footer section-pad">
-  <h3 class="v-pad-bottom text-center">Our Sponsors</h3>
+  <h3 class="v-pad-bottom text-center">Sponsors</h3>
+      <div class="row v-pad-top">
+        <div class="medium-6 column medium-centered text-center">
+        <p>Interested in becoming a sponsor? There's still time!</p>
+        <a href="/sponsors/information/" class="button">Become a Sponsor</a>
+      </div>
+        </div>
```

Comment:
```suggestion
    <div class="row v-pad-top">
      <div class="medium-6 column medium-centered text-center">
        <p>Interested in becoming a sponsor? There's still time!</p>
        <a href="/sponsors/information/" class="button">Become a Sponsor</a>
      </div>
    </div>
```

Minor indentation fix here